bucket/confuser-ex.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.5.0",
+    "description": "A .NET obfuscator",
```

Comment:
Still same issue.. Completely nonsense for end user

Reply:
It's a .NET IL obfuscator, like proguard for java, to make reverse engineering of distributed binaries hard, it's for pro developer's only and I think the word obfuscator is clear enough in this context, what do you think?